### ISSUE TITLE
Play embedded youtube videos within the docs

### ISSUE THREAD

**Issue by yahiheb** [zkSNACKs/WasabiDoc#1394]
https://github.com/zkSNACKs/WasabiDoc/pull/1247#issuecomment-1339747268
https://github.com/zkSNACKs/WasabiDoc/pull/1247#issuecomment-1342209641

Possible solution: https://github.com/vuepress/vuepress-next/discussions/1182

**Comment by jesseaam** [comment]
>Possible solution: https://github.com/vuepress/vuepress-next/discussions/1182

I think this would be a great solution, though I've been having trouble getting it to work. When you install the plugin `vuepress-plugin-components@next` it causes the docs not to show up anymore.

### Documentation

<details>
<summary>Running the current master this is what you see when you start the development server:
</summary>

```
npm start

> wasabidoc@0.1.0 prestart
> rm -rf $npm_package_config_dist


> wasabidoc@0.1.0 start
> vuepress dev docs

warning an anonymous plugin or theme was detected and ignored
warning an anonymous plugin or theme was detected and ignored
warning an anonymous plugin or theme was detected and ignored
✔ Initializing and preparing data - done in 986ms

  vite v3.1.8 dev server running at:

  ➜  Local:   http://localhost:8080/
```
</details>

<details>
<summary>Now install the plugin:</summary>

```
$ npm i -D vuepress-plugin-components@next

added 326 packages, removed 2 packages, changed 24 packages, and audited 795 packages in 21s

78 packages are looking for funding
  run `npm fund` for details

6 vulnerabilities (2 moderate, 4 high)

To address issues that do not require attention, run:
  npm audit fix

Some issues need review, and may require choosing
a different dependency.

Run `npm audit` for details.
```
</details>


<details>
<summary>Running the current master AFTER INSTALLING PLUGIN this is what you see when you start the development server:
</summary>

```
$ npm start

> wasabidoc@0.1.0 prestart
> rm -rf $npm_package_config_dist


> wasabidoc@0.1.0 start
> vuepress dev docs

warning an anonymous plugin or theme was detected and ignored
warning an anonymous plugin or theme was detected and ignored
warning an anonymous plugin or theme was detected and ignored
✔ Initializing and preparing data - done in 1.19s

  vite v4.0.0 dev server running at:

  ➜  Local:   http://localhost:8080/
```
</details>

In short the difference you see is before and after installing the plugin:
*  vite v3.1.8 dev server running at:
*  vite v4.0.0 dev server running at:

---

I started a discussion [here](https://github.com/vuepress-theme-hope/vuepress-theme-hope/discussions/2538) on GitHub Discussions, so maybe the community there will have some tips.

**Comment by MarnixCroes** [comment]
What's the status of this?
@CAnorbo any ideas?
